tests/fixtures/tests.py

```diff
@@ -342,6 +342,15 @@ def test_loaddata_app_option(self):
             '<Article: Who needs more than one database?>',
         ])
 
+    def test_loaddata_verbose(self):
+        # Load db fixture 1 with high verbosity level
+        new_io = six.StringIO()
+        management.call_command('loaddata', 'fixture1.json', verbosity=3, stdout=new_io, stderr=new_io)
+        command_output = new_io.getvalue().strip()
+
+        # Check the output, that all objects were processed
+        self.assertIn("Processed 1 object(s).\nProcessed 2 object(s).\nProcessed 3 object(s).\nProcessed 4 object(s).\n", command_output)
```

Comment:
Use a style like this to avoid long lines:
```
self.assertEqual(
    "Processed 1 object(s).\nProcessed 2 object(s).\n"
    "Processed 3 object(s).\nProcessed 4 object(s).\n",
    command_output
)
```